pages/clubs.jsx

```diff
@@ -18,6 +18,29 @@ var ga = require('react-ga');
 
 var Illustration = require('../components/illustration.jsx');
 
+var PlaceSquare = React.createClass({
+  NUM_HASH_COLORS: 6,
+  hashColor: function() {
+    var loc = this.props.location;
+    var sum = 0;
+
+    for (var i = 0; i < loc.length; i++) {
+      sum += loc.charCodeAt(i);
+    }
+
+    return 'hash-color-' + (sum % this.NUM_HASH_COLORS);
+  },
+  render: function() {
+    var abbr = this.props.location.slice(0, 3);
```

Comment:
This won't actually generate country or city codes, just so you know.  As an example, Vancouverites proudly wear the YVR airport code.